### DISCUSSION TITLE
请求有横扫千星这款游戏

### DISCUSSION THREAD

**Discussion by MAMAYYY** [Gamuxorg/bbs#293] [投递和请求游戏]
游戏名称：横扫千星
游戏名称(英文)：Planetary Annihilation
下载地址 :（如果有的话）：

**Comment by hyh19962008** [comment]
这个游戏感觉就是各种海就对了，对面也是海，然后因为资源无限，刚换了一波又造了下一波出来。

**Comment by hyh19962008** [comment]
已添加

**Comment by MAMAYYY** [comment]
谢谢你，感谢

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: ***@***.***>
发送时间: 2022年3月24日 9:24
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [Gamuxorg/bbs] 请求有横扫千星这款游戏 (Issue #180)


已添加

―
